### ISSUE TITLE
TTS objective evaluation

### ISSUE THREAD

**Issue by sw005320** [espnet/espnet#5495]
Compared with the ASR and other templates, the current TTS template does not include the objective evaluation stage even if we prepare various evaluation scripts.
Due to the current issue, the TTS recipe README.md only includes the model links and we’re not sure which one we should pick up.

Of course, it requires vocoder training in some cases, but it is also worth considering to include in the TTS template to make the TTS recipe more concrete.

**Comment by sw005320** [comment]
@wyh2000, do you think you can incorporate this to your project (https://github.com/espnet/espnet/issues/5475)?

Actually, your recipe (https://github.com/espnet/espnet/tree/master/egs2/bibletts/tts1) includes the objective evaluation and it would be a good start.

**Comment by ftshijt** [comment]
We introduce a specific stage for svs, I guess we could use similar style for TTS as well

https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/svs1/svs.sh#L1067-L1116

**Comment by wyh2000** [comment]
Yes, I can incorporate it into voice conversion task and follow svs task's style.
